roadmap.md

```diff
@@ -0,0 +1,42 @@
+# Optimistic Roadmap
+
+![Roadmap Diagram](./assets/roadmap.svg)
+
+## Design Goals
+
+The roadmap & abstractions are designed to enable the independent development of each component. The 4 major components are: 
+1. the optimistic mainnet deployment,
+2. the fraud proof infrastructure,
+3. stateless clients, 
+4. sharding. 
+
+Each component will produce incremental and independent releases, each driving closer to unification and Optimistic Ethereum nirvana.
+
+## Summary of each component
+
+### Optimism Mainnet
+
+Optimism mainnet is the first Optimistic Ethereum network. It serves as:
+
+1. A testbed for protocol changes; and
```

Comment:
I think selling L2s as a testbed for protocol changes is slippery slope. As L2s accrue value, I expect users will become more and more cautious of changes.

Reply:
Agreed 100%. Updated the language! This is going to have to be hella stable, you are 100% correct fam.